rasa/server.py

```diff
@@ -69,7 +69,7 @@
 from rasa.core.utils import AvailableEndpoints
 from rasa.nlu.emulators.no_emulator import NoEmulator
 from rasa.nlu.test import run_evaluation, CVEvaluationResult
-from rasa.utils.endpoints import EndpointConfig
+from rasa.utils.endpoints import EndpointConfig, bool_arg, float_arg, int_arg
```

Comment:
we have some internal code convention where we say that don't want to import functions directly in favor of explictly referring to it by module. So it would be great if you could also refer to `bool_arg` etc. using `rasa.utils.endpoints.bool_arg` 🙌🏻

Reply:
Done